plugins/modules/xfconf.py

```diff
@@ -195,15 +195,12 @@ class XFConfProperty(StateModuleHelper):
 
     default_state = 'present'
 
-    def update_xfconf_output(self, **kwargs):
-        self.update_vars(meta={"output": True, "fact": True}, **kwargs)
-
     def __init_module__(self):
         self.runner = xfconf_runner(self.module)
         self.does_not = 'Property "{0}" does not exist on channel "{1}".'.format(self.vars.property,
                                                                                  self.vars.channel)
-        self.vars.set('previous_value', self._get(), fact=True)
-        self.vars.set('type', self.vars.value_type, fact=True)
+        self.vars.set('previous_value', self._get())
+        self.vars.set('type', self.vars.value_type)
```

Comment:
Strictly speaking, yes, but we had deprecated facts before in https://github.com/ansible-collections/community.general/commit/af64c9a432417b6694ae936fb5566fcfba763786

This code lingered there by mistake. The documentation indicates that we do not provide facts anymore. IMO it should be safe enough for removal, but if you want to start the deprecation process again, we can do it.

Reply:
The option `disable_facts` still exists and can be set to `false`. If I understand this correctly, we have to deprecate that function first and when removing it also remove this.

Reply:
Actually, the if block at line 206 below raises an exception if disable_facts is false.

Reply:
Does `fact=True` say that these variables are returned both as regular return variables *and* as `ansible_facts`?

Reply:
It has no effect whatsoever on the return variables. This flag is used strictly to indicate this will be returned as a fact. There is another flag, `output`, to control whether the values will be exposed as return variables or not. That one is `True` by default, so that's why it is not being set.